tasks/postgresql.yml

```
@@ -4,10 +4,12 @@
     - name: Configure postgresql repository key
       apt_key:
         url: "{{ postgresql_apt_key_url }}"
+      when: postgresql_apt_key_url != ""
 
     - name: Configure postgresql repository
       apt_repository:
         repo: "{{ postgresql_apt_repo }}"
+        filename: "{{ postgresql_apt_repo_name | default(omit) }}"

```

```suggestion
        filename: "{{ postgresql_apt_filename }}"

```
I'd change the `postgresql_apt_repo_name` variable name to `postgresql_apt_filename` for consistency with apt_repository module properties.

And one more thing - it would be great to set this variable to any default value and keep it in `vars/Debian.yml`. Since by default the pgdg apt repo creates the `/etc/apt/sources.list.d/apt_postgresql_org_pub_repos_apt.list` file. My suggestion is introducing an additional variable `__postgresql_apt_filename` and setting it to the `apt_postgresql_org_pub_repos_apt` value. What do you think?

---

No problem. I just updated the requested changes.